packages/frontend-2/components/viewer/selection/Sidebar.vue

```diff
@@ -28,6 +28,17 @@
             Isolate
           </div>
         </FormButton>
+        <div class="w-full text-right">
+          <FormButton
```

Comment:
Maybe a tooltip here? @benjaminvo

Reply:
Sorry I didn't see this comment. Yes I agree! But let's just follow up with that and merge this now so we don't need to wait for the CI checks again.